src/types/supabase.ts

```diff
@@ -0,0 +1,106 @@
+export type Json =
```

Comment:
These types are great to have! Hope you didn't type them manually ;) I imagine you used the [Supabase CLI for generating TS types](https://supabase.com/docs/guides/database/api/generating-types). So I would add a prep script to our package to automate this and keep this in sync in the future.